### ISSUE TITLE
Running into apt-get problems in Shippable

### ISSUE THREAD

**Issue by mikkopori** [Shippable/support#3807]
Hello!

Locally I'm fine, but updating a changed project causes some issues:

https://app.shippable.com/github/mikkopori/enterlot/runs/609/1/console
```

W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://deb.opera.com/opera stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D615560BA5C7FF72
W: Failed to fetch http://deb.opera.com/opera/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D615560BA5C7FF72
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists...
Building dependency tree...
Reading state information...
build-essential is already the newest version (12.1ubuntu2).
default-jdk is already the newest version (2:1.8-56ubuntu2).
python3 is already the newest version (3.5.1-3).
python3 set to manually installed.
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 npm : Depends: nodejs but it is not going to be installed
       Depends: node-abbrev (>= 1.0.4) but it is not going to be installed
       Depends: node-ansi (>= 0.3.0-2) but it is not going to be installed
       Depends: node-ansi-color-table but it is not going to be installed
       Depends: node-archy but it is not going to be installed
       Depends: node-block-stream but it is not going to be installed
       Depends: node-fstream (>= 0.1.22) but it is not going to be installed
       Depends: node-fstream-ignore but it is not going to be installed
       Depends: node-github-url-from-git but it is not going to be installed
       Depends: node-glob (>= 3.1.21) but it is not going to be installed
       Depends: node-graceful-fs (>= 2.0.0) but it is not going to be installed
       Depends: node-inherits but it is not going to be installed
       Depends: node-ini (>= 1.1.0) but it is not going to be installed
       Depends: node-lockfile but it is not going to be installed
       Depends: node-lru-cache (>= 2.3.0) but it is not going to be installed
       Depends: node-minimatch (>= 0.2.11) but it is not going to be installed
       Depends: node-mkdirp (>= 0.3.3) but it is not going to be installed
       Depends: node-gyp (>= 0.10.9) but it is not going to be installed
       Depends: node-nopt (>= 3.0.1) but it is not going to be installed
       Depends: node-npmlog but it is not going to be installed
       Depends: node-once but it is not going to be installed
       Depends: node-osenv but it is not going to be installed
       Depends: node-read but it is not going to be installed
       Depends: node-read-package-json (>= 1.1.0) but it is not going to be installed
       Depends: node-request (>= 2.25.0) but it is not going to be installed
       Depends: node-retry but it is not going to be installed
       Depends: node-rimraf (>= 2.2.2) but it is not going to be installed
       Depends: node-semver (>= 2.1.0) but it is not going to be installed
       Depends: node-sha but it is not going to be installed
       Depends: node-slide but it is not going to be installed
       Depends: node-tar (>= 0.1.18) but it is not going to be installed
       Depends: node-underscore but it is not going to be installed
       Depends: node-which but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```
Should I be using some other image than:

`build:
  pre_ci:
  pre_ci_boot:
    image_name: drydock/u16pytall
    image_tag: v5.4.1
    options: "-p 5432:5432 -p 8888:8888 --net=bridge"
`

### Description of your issue:
NOTE: If you are experiencing a build failure, please include:

- Link to the build failure
- Scripts called from the yml (optional)

**Comment by ambarish2012** [comment]
@mikkopori I suggest you try some suggestions here - https://askubuntu.com/questions/223237/unable-to-correct-problems-you-have-held-broken-packages.

**Comment by mikkopori** [comment]
Hello,

updated the image version to the latest according to this: http://docs.shippable.com/platform/runtime/language/python/

and that helped with the opera-issue:

W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://deb.opera.com/opera stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D615560BA5C7FF72
W: Failed to fetch http://deb.opera.com/opera/dists/stable/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D615560BA5C7FF72

Also changed the relations between installing nodejs and npm, which seemed to solve the problem.

Tnx!
